src/stdlib/WebSharper.JQuery/Definition.fs

```diff
@@ -323,42 +349,42 @@ module Definition =
             Generic - fun t -> "prop" => T<string> ^-> t
             Generic - fun t -> "prop" => T<string> * t ^-> TSelf
             "prop" => T<obj> ^-> TSelf
-            "prop" => T<string> * (T<int> * T<obj> ^-> T<obj>) ^-> TSelf
+            "prop" => T<string> * (T<obj> + (T<int> * T<obj> ^-> T<obj>)) ^-> TSelf
             "removeAttr" => T<string> ^-> TSelf
             "removeClass" => T<string> ^-> TSelf
             "removeClass" => (T<int> * T<obj> ^-> T<string>) ^-> TSelf
             "removeProp" => T<string> ^-> TSelf
-            "toggleClass" => T<string> ^-> TSelf
-            "toggleClass" => T<string> * T<bool> ^-> TSelf
-            "toggleClass" => (T<int> * T<string> * T<bool> ^-> T<string>) * !?T<bool> ^-> TSelf
+            "toggleClass" => T<string> + !| T<string> ^-> TSelf
+            "toggleClass" => (T<string> + !| T<string>) * T<bool> ^-> TSelf
+            "toggleClass" => (T<int> * T<string> * T<bool> ^-> T<string> + !| T<string>) * !?T<bool> ^-> TSelf
             "val" => T<unit> ^-> T<obj>
             "val" => (T<string> + T<float> + T<obj []>) ^-> TSelf
             "val" => (T<int> * T<string> ^-> T<string>) ^-> TSelf
 
             // CSS related instance methods
             "css" => T<unit> ^-> T<string>
-            "css" => (T<string> + T<string []>) ^-> T<string>
-            "css" => T<string> * T<string> ^-> TSelf
-            "css" => T<string> * (T<int> * T<string> ^-> T<string>) ^-> TSelf
+            "css" => (T<string> + !| T<string>) ^-> T<string>
+            "css" => T<string> * (T<string> + T<int>) ^-> TSelf
+            "css" => T<string> * (T<int> * T<string> ^-> T<string> + T<int>) ^-> TSelf
             "css" => T<Object<string>> ^-> TSelf
             "height" => T<unit> ^-> T<int>
             "height" => T<int> ^-> TSelf
-            "height" => (T<int> * T<int> ^-> T<int>) ^-> TSelf
+            "height" => (T<int> * T<int> ^-> T<string> + T<int>) ^-> TSelf

```

Comment:
the *height and *width functions seem to be missing their string overrides on their setters.